openff/toolkit/topology/molecule.py

```diff
@@ -1163,6 +1170,27 @@ def to_dict(self) -> dict:
         molecule_dict
             A dictionary representation of the molecule.
 
+            - **name** (str): An optional name to be associated with the molecule
+            - **atoms** (list[dict]): A list of dictionary inputs for :meth:`Atom.from_dict()`
+            - **bonds** (list[dict]): A list of dictionary inputs for :meth:`Bond.from_dict()`
+            - **conformers** (list[list]): A list containing the cartesian coordinates of each atoms in the order
```

Comment:
It'd be useful to specify units / reference `conformers_unit` here, since they're lost in the downcast `Quantity` -> `list[list[float]]`, sinc

Reply:
Yep exactly